### ISSUE TITLE
ListCollaborators get Exception: You must set one case of the following fields in your request proto: ['user_app_id.user_id', 'user_app_id.app_id']

### ISSUE THREAD

**Issue by frizfealer** [Clarifai/clarifai-python-grpc#14]
When I use API call ListCollaborators, given user_app_id.user_id', 'user_app_id.app_id', I still get the exception that these fields are not given.
Here is my code
```
res = stub.ListCollaborators(
    ListCollaboratorsRequest(user_app_id=UserAppIDSet(user_id=USER_ID, app_id=APP_ID)), 
    metadata=auth_metadata)
```
The error I got is:
```
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
<ipython-input-20-6bdd86034aee> in <module>
----> 1 stub.ListCollaborators(ListCollaboratorsRequest(user_app_id=UserAppIDSet(user_id=user_id, app_id='labeling-Windows-and-Doors')), metadata=auth_metadata)

~/anaconda3/envs/data_science_local/lib/python3.8/site-packages/clarifai_grpc/channel/grpc_json_channel.py in __call__(self, request, metadata)
    169         params = protobuf_to_dict(request, use_integers_for_enums=False, ignore_show_empty=True)
    170 
--> 171         url, method, url_fields = _pick_proper_endpoint(self.resources, params)
    172 
    173         for url_field in url_fields:

~/anaconda3/envs/data_science_local/lib/python3.8/site-packages/clarifai_grpc/channel/grpc_json_channel.py in _pick_proper_endpoint(resources, request_dict)
    298 
    299     if not best_match_url:
--> 300         raise Exception(
    301             "You must set one case of the following fields in your request proto: "
    302             "%s" % all_fields

Exception: You must set one case of the following fields in your request proto: ['user_app_id.user_id', 'user_app_id.app_id']
```

The auth_metadata and user_id, and app_id works on other operation, e.g. GetInputCountRequest. So I am wondering if there a bug in this function, instead of my setting of metadata error.

Thanks,
Yeu-Chern

**Comment by rok-clarifai** [comment]
@frizfealer Thank you for informing us of the problem. We've reproduced the issue and found a fix. A fixed version will be published shortly.

**Comment by rok-clarifai** [comment]
@frizfealer We've released a new version 7.0.1 that has a fix. Please let us know if you encounter any more issues.

Note however that the issue manifested only when using the JSON channel (via `ClarifaiChannel.get_json_channel()`). We definitely recommend using the gRPC channel directly (via `ClarifaiChannel.get_grpc_channel()`) since there are basically no downsides.
